print/layouts/default.vue

```diff
@@ -76,3 +76,15 @@ useHead(header)
   padding: 0;
 }
 </style>
+
+<!--
+Here we need to hack away 2 divs which nuxt generates,
+so these styles must be global.
+-->
+<!-- eslint-disable-next-line vue-scoped-css/enforce-style-type -->
+<style lang="scss">
+body > div > div.container > div:not(:has(*)),
```

Comment:
This selector seems quite broad... Would this hide an empty div with fixed width / height and some borders / bg color? (Such as I am using in client print to render the dotted lines on the toc)